### PR TITLE
Steps not properly configured

### PR DESCRIPTION
**Pull request by ct-goranpalibrk**
Brief description of the PR.
Changed error message when steps in jobs are not properly configured.

**Related Issue**
This PR fixes issue #2475

**Description of the solution adopted**
A more detailed description of the changes made to solve/close one or more issues.
If the PR is simple and easy to inderstand this section can be skipped.

**Screenshots**
If applicable, add screenshots to help explain your solution

**Any side note on the changes made**
Description of any other change that has been made, which is not directly linked to the issue resolution
[e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR successfully resolves issue #2475.

**Comment by muros-ct** [comment]
@lorthirk  can you confirm that adding new dependency is OK. I think that there is no other option regarding this error handling. Either we add dependency to specific UI module or we add specific error messages regarding this module, to console messages.

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/server/util/KapuaExceptionHandler.java [hunk 1]
```diff
@@ -148,6 +149,8 @@ public static void handle(Throwable t) throws GwtKapuaException {
             throw new GwtKapuaException(GwtKapuaErrorCode.RETRY_AND_CRON_BOTH_SELECTED, t, t.getLocalizedMessage());
         } else if (t instanceof KapuaException && ((KapuaException) t).getCode() == (KapuaErrorCodes.TRIGGER_NEVER_FIRE)){
             throw new GwtKapuaException(GwtKapuaErrorCode.TRIGGER_NEVER_FIRE, t, t.getLocalizedMessage());
+        } else if (t instanceof KapuaException && ((KapuaException) t).getCode() == (KapuaJobEngineErrorCodes.JOB_STARTING_ERROR)){
```
Comment (Coduz) [inline console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/server/util/KapuaExceptionHandler.java]: Please match on the String value of the ErrorCode.

File: console/module/api/pom.xml [hunk 2]
```diff
@@ -29,6 +29,10 @@
             <groupId>org.eclipse.kapua</groupId>
             <artifactId>kapua-security-shiro</artifactId>
         </dependency>
+         <dependency>
```
Comment (Coduz) [inline console/module/api/pom.xml]: Drop this dependency

Comment (ct-goranpalibrk) [reply console/module/api/pom.xml]: @Coduz dependency removed.

**Comment by ct-goranpalibrk** [comment]
@Coduz @LeoNerdoG PR changes are implemented. Please take a look.

**Comment by LeoNerdoG** [comment]
I have verified that this is now OK from GUI standpoint.

**Comment by Coduz** [comment]
Hi @LeoNerdoG 

can you please recheck after force pushing the code? 

I see that the error code has changed only on the `*-messages.properties` file but nowhere else...

**Comment by LeoNerdoG** [comment]
Hi @Coduz,
yes, I can confirm that after the latest changes the issue is still not fixed. I have already contacted Goran - he is already working on it. 

Screenshot 2019-04-01 at 08 57 58

**Comment by ct-goranpalibrk** [comment]
Hi @Coduz , @LeoNerdoG  Please check is everything is ok after last force-push.

**Comment by LeoNerdoG** [comment]
Hey,
I have tried this and now I cannot even login 😸 

Screenshot 2019-04-01 at 13 04 10

**Comment by LeoNerdoG** [comment]
Hi,
I have retested this and now I get this error message, which is OK: 

Screenshot 2019-04-01 at 15 12 13

Should we merge this?

**Pull request merged**